name_node/models.go

```diff
@@ -0,0 +1,16 @@
+package namenode
+
+import (
+	"time"
+
+	"github.com/jinzhu/gorm"
+)
+
+// Clip Represents a clip record in the ingestion database
+type Clip struct {
+	gorm.Model
+	Token     string    //The token of the video to which this clip belongs
+	Tag       string    //The classification of the clip
+	StartTime time.Time //Start time of the clip's tag
+	EndTime   time.Time //End time for the clip's tag
```

Comment:
I don't think the type time is suitable for this case, as time represents a date.
@Kareem-Emad what is most suitable representation?

Reply:
We agreed to use big int (secs from start).